File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.js
function solution(my_string) {
    var answer = '';
    
    Array.from(my_string).forEach((cur) => {
        if(isupper(cur)){
            answer += (cur.toLowerCase());
        }else{
            answer += (cur.toUpperCase());
        }
    })
    
    
    return answer;
}


function isupper(s){
    return s === s.toUpperCase();
}

function islower(s){
    return s === s.toLowerCase();
}

File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.java
import java.util.Arrays;
import java.util.stream.Collectors;

class Solution {
    public String solution(String my_string) {
        return Arrays.stream(my_string.split(""))
                     .map(cur -> {
                         if (cur.equals(cur.toLowerCase())) {
                             return cur.toUpperCase();
                         } else {
                             return cur.toLowerCase();
                         }
                     })
                     .collect(Collectors.joining());
    }
}